src/diffusers/loaders/lora.py

```diff
@@ -137,13 +152,12 @@ def lora_state_dict(
         **kwargs,
     ):
         r"""
-        Return state dict for lora weights and the network alphas.
+        Return state dict and network alphas of the LoRA weights.
 
         <Tip warning={true}>
 
-        We support loading A1111 formatted LoRA checkpoints in a limited capacity.
-
-        This function is experimental and might change in the future.
+        A1111 formatted LoRA checkpoints are supported in a limited capacity. This function is experimental and might
+        change in the future.
```

Comment:
We can safely remove this warning now. We have significantly the support for loading non-diffusers LoRA checkpoints over the past few months. @patrickvonplaten WDYT?